ungrib/src/ngl/g2/intmath.f

```diff
@@ -204,7 +204,7 @@ function ilog2_1(i_in)
       ilog2_1=0
       i=i_in
       if(i<=0) return
-      if(iand(i,i-1)/=0) then
+      if(iand(i,i-1_1)/=0) then
          !write(0,*) 'iand i-1'
          ilog2_1=1
```

Comment:
Similarly here, do we say
```
ilog2_1=1_1
```

Reply:
There should be no problem here to let the compiler perform the type conversion.